ui/components/app/confirm/shared/info/row/value-double.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Box, Text } from '../../../../../component-library';
+import {
+  AlignItems,
+  Display,
+  FlexDirection,
+  TextColor,
+  FlexWrap,
+} from '../../../../../../helpers/constants/design-system';
+
+export type ConfirmInfoRowValueDoubleProps = {
+  left: string;
+  right: string;
+};
+
+export const ConfirmInfoRowValueDouble = ({
+  left,
+  right,
+}: ConfirmInfoRowValueDoubleProps) => (
+  <Box
```

Comment:
would it make sense to use ConfirmInfoRow for all of the row variants so that they can also support the states?

this comment applies for address and value-double rows

Reply:
This should be solved now with the context!